### DISCUSSION TITLE
Version 1.23.1

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#288] [Announcements]
- Fix favorites unintentionally disappearing from favorites widget
- Fix crash

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.23.1'>Version 1.23.1</a>.</em>

**Comment by sohan-f** [comment]
sir can you bring a version for android 7, 32bit arch?

**Reply by MM2-0** [reply]
Certainly not.

**Comment by ogundijitiwalope** [comment]
It's perfect, I absolutely love it 😄🙂
Damn can't wait for another update.
The only issue I have with it is that it's a bit choppy

**Comment by SoaibHaque** [comment]
I love this update but when I open any app it takes much more time than usual.
